torchbenchmark/models/yolov3/yolo_models.py

```diff
@@ -103,13 +103,13 @@ def create_modules(module_defs, img_size, cfg):
                                 layers=layers,  # output layers
                                 stride=stride[yolo_index])
 
-            # Initialize preceding Conv2d() bias (https://arxiv.org/pdf/1708.02002.pdf section 3.3)
             try:
                 j = layers[yolo_index] if 'from' in mdef else -1
                 bias_ = module_list[j][0].bias  # shape(255,)
                 bias = bias_[:modules.no * modules.na].view(modules.na, -1)  # shape(3,85)
-                bias[:, 4] += -4.5  # obj
-                bias[:, 5:] += math.log(0.6 / (modules.nc - 0.99))  # cls (sigmoid(p) = 1/nc)
+                with torch.no_grad():  # avoids "requires grad is being used in an in-place operation"
```

Comment:
i think this is fine, but since it could affect perf i'd not want to merge this part to the 0.1 branch

Reply:
This code is part of the setup so shouldn't effect perf